usr/config.h

```diff
@@ -199,6 +199,7 @@ typedef struct session_rec {
 	char					boot_root[BOOT_NAME_MAXLEN];
 	char					boot_nic[BOOT_NAME_MAXLEN];
 	char					boot_target[BOOT_NAME_MAXLEN];
+	int                                     reopen_log_freq;
```

Comment:
ok, it's int64_t above (which seemed excessive). Since you're using int here, how about int above?

Reply:
The above needs int64_t as mentioned by me in the above comment. Here it needs int as the function idbm_recinfo_node() in usr/idbm.c calls __recinfo_int() which needs int.